src/lib/journey/callbacks/choice/choice.svelte

```diff
@@ -37,13 +42,26 @@
   }
 </script>
 
-<!-- TODO: Needs a localization strategy -->
-<Select
-  {firstInvalidInput}
-  defaultOption={defaultChoice}
-  isRequired={false}
-  key={inputName}
-  label={prompt}
-  onChange={setValue}
-  options={choiceOptions}
-/>
+{#if displayType === 'select'}
+  <!-- TODO: Needs a localization strategy -->
+  <Select
+    {firstInvalidInput}
+    defaultOption={defaultChoice}
+    isRequired={false}
+    key={inputName}
+    label={prompt}
+    onChange={setValue}
+    options={choiceOptions}
+  />
+{:else}
+  <Radio
+    {firstInvalidInput}
+    defaultOption={Number(defaultChoice)}
+    isRequired={false}
+    key={inputName}
+    message={prompt}
+    onChange={setValue}
+    name={inputName}
+    options={choiceOptionsForRadio}
```

Comment:
See above comment about the `choiceOptions`

Reply:
Let's fix the type bug, so we don't have to do this. Let me know if that makes sense.